runtime/src/iree/hal/drivers/hip/hip_device.c

```diff
@@ -154,9 +299,26 @@ static iree_status_t iree_hal_hip_device_create_internal(
   device->hip_dispatch_stream = dispatch_stream;
   device->host_allocator = host_allocator;
 
-  iree_status_t status = iree_hal_hip_pending_queue_actions_create(
-      symbols, hip_device, &device->block_pool, host_allocator,
-      &device->pending_queue_actions);
+  iree_hal_hip_deferred_work_queue_device_interface_t* symbol_table;
```

Comment:
this should be `device_interface` instead of `symbol_table` (maybe old renaming?)